src/cov.jl

```diff
@@ -154,6 +154,15 @@ function cor2cov!(C::AbstractMatrix, s::AbstractArray)
     return C
 end
 
+"""
+    cov2cor!(C)
+
+Compute the correlation matrix from the covariance matrix `C`, in-place.
+
+The leading diagonal is used to determine the standard deviations by which to normalise.
+"""
+cov2cor!(C::AbstractMatrix) = cov2cor!(C, sqrt.(diag(C)))
```

Comment:
This should probably extend the existing `Statistics.cov2cor!`, but that'd be type piracy. Maybe we can leave this out or call it `_cov2cor!` here and add it as a method to the Statistics.jl stdlib?

Reply:
Yep good point! There's some relevant discussion here it seems: https://github.com/JuliaStats/StatsBase.jl/issues/444

For now I'll just rename to be a private method, to avoid getting bogged down in this issue.

Reply:
(done)